pkg/api/v1alpha1/cloudstackdatacenterconfig_types.go

```diff
@@ -50,6 +48,16 @@ const (
 	Name CloudStackResourceRefType = "Name"
 )
 
+// Zone is an organizational construct typically used to represent a single datacenter, and all its physical and virtual resources exist inside that zone. It can either be specified as a UUID or name
+type Zone struct {
+	// ZoneRef is the name or UUID of the CloudStack zone in which clusters should be created. Zones should be managed by a single CloudStack Management endpoint.
+	ZoneRef CloudStackResourceRef `json:"zone"`
```

Comment:
Adding "ref" suffix here makes it inconsistent I think. If we add the prefix to Zone above, then we can change ZoneRef -> Zone